src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/MultiPloidyGenotyper.java

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.hellbender.tools.walkers.haplotypecaller;
+
+import htsjdk.samtools.util.Locatable;
+import htsjdk.samtools.util.OverlapDetector;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.SimpleCount;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.GenotypingEngine;
+
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.IntFunction;
+
+public class MultiPloidyGenotyper<T extends GenotypingEngine> {
```

Comment:
docs

Reply:
also consider naming this to be a little more clear. This is not a genotyper this is a ploidy switch for genotypers

Reply:
Yeah, I was hoping someone else would name it :)